public/apps/account/test/plugin.test.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ *   Copyright OpenSearch Contributors
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import { interceptError } from '../../../utils/logout-utils';
+import { setShouldShowTenantPopup } from '../../../utils/storage-utils';
+import { LOGIN_PAGE_URI } from '../../../../common';
+
+jest.mock('../../../utils/storage-utils', () => ({
+  setShouldShowTenantPopup: jest.fn(),
+}));
+
+describe('Session timeout', () => {
+  const fakeError = {
+    response: {
+      status: 401,
+    },
+  };
+
+  it('Should call setShouldShowTenantPopup on session timeout', () => {
+    const sessionTimeoutFn = interceptError(LOGIN_PAGE_URI, window);
+    sessionTimeoutFn(fakeError, null);
+    expect(setShouldShowTenantPopup).toBeCalledTimes(1);
```

Comment:
It would also be good to test a negative test cases and be sure the popup isn't called on the non 401 case

Reply:
Thx for the suggestion @peternied! I have updated it with a negative test cases for this one.